api/all/src/main/java/io/opentelemetry/api/DefaultOpenTelemetry.java

```diff
@@ -17,11 +17,11 @@
 final class DefaultOpenTelemetry implements OpenTelemetry {
   private static final OpenTelemetry NO_OP = new DefaultOpenTelemetry(ContextPropagators.noop());
 
-  static OpenTelemetry noop() {
+  static OpenTelemetry getNoop() {
```

Comment:
Had to rename these due to errorprone, which is presumably overstrict on these private APIs

```
/Users/aanuraag/git/opentelemetry-java/api/all/src/main/java/io/opentelemetry/api/DefaultOpenTelemetry.java:26: warning: [FunctionalInterfaceClash] When passing lambda arguments to this function, callers will need a cast to disambiguate with:
  static OpenTelemetry propagating(ContextPropagators propagators) {
                       ^
      OpenTelemetry.propagating(ContextPropagators)
    (see https://errorprone.info/bugpattern/FunctionalInterfaceClash)
error: warnings found and -Werror specified
```

Reply:
Internal API, I don't care :)